Formula/s/scarb.rb

```diff
@@ -24,7 +25,15 @@ class Scarb < Formula
   depends_on "rust" => :build
 
   def install
-    system "cargo", "install", *std_cargo_args(path: "scarb")
+    %w[
+      scarb
+      extensions/scarb-cairo-language-server
+      extensions/scarb-cairo-run
+      extensions/scarb-cairo-test
+      extensions/scarb-snforge-test-collector
+    ].each do |f|
```

Comment:
Is it possible to find them dynamically, or should we just keep this in sync if the list changes in future Scarb releases? Would removing one extension cause an error or just a warning?

Reply:
it would cause error, treat this as kind of binary spec